docs/roadmap.rst

```diff
@@ -0,0 +1,223 @@
+AFNI users at this point are therefore not accessing the latest version of ``tedana``.
+We will grow our user base if ``tedana`` can be accessed through AFNI,
+and we are therefore committed to supporting native integration of ``tedana`` in AFNI.
+
+.. _afni_proc.py: https://afni.nimh.nih.gov/pub/dist/doc/program_help/afni_proc.py.html
+.. _tedana_wrapper.py: https://github.com/afni/afni/blob/a3288abefb66bc7c76e98fdf13425ab48651bf36/src/python_scripts/afni_python/tedana_wrapper.py
+
+One metric of success, therefore, will be if we can demonstrate sufficient stability and support
+such that the ``afni_proc.py`` maintainers are willing to switch to ``tedana`` as the recommended
+method of accessing ME-EPI denoising in AFNI.
+We will aim to aid in this process by increasing compatibility between ``tedana``
+and the ```afni_proc.py`` workflow, eliminating the need for an additional wrapper script.
+For example, ``tedana`` could directly accept BRIK/HEAD files,
+facilitating interoperability with other AFNI pipelines.
+
+**`Associated Milestone`_**
+
+.. _Associated Milestone: https://github.com/ME-ICA/tedana/milestone/8
+
+This milestone will close when ``tedana`` is stable enough such that the recommended default in
+ ``afni_proc.py`` is to access ME-EPI denoising via ``pip install tedana``,
+ rather than maintaining the alternative version that is currently used.
+
+
+Workflow integration: BIDS
+``````````````````````````
+**Summary**:
+Currently, the BIDS ecosystem has limited support for ME-EPI processing.
+We will grow our user base if ``tedana`` is integrated into existing BIDS Apps and
+therefore accessible to members of the BIDS community.
+One promising opportunity is if ``tedana`` can be used natively in `FMRIPrep`_.
+Some of the work is not required at this repository, but other changes will need to happen here;
+for example, making sure the outputs are BIDS compliant.
+
+A metric of success, then, will be
+1. Fully integrating ``tedana`` into ``FMRIPrep``, and
+2. Making ``tedana`` outputs compliant with the `BIDS derivatives specification`_.
+
+.. _FMRIPrep: https://github.com/poldracklab/fmriprep
+.. _BIDS derivatives specification: https://docs.google.com/document/d/1Wwc4A6Mow4ZPPszDIWfCUCRNstn7d_zzaWPcfcHmgI4/edit
+
+**`Associated Milestone`_**
+
+.. _Associated Milestone: https://github.com/ME-ICA/tedana/milestone/9
+
+This milestone will close when the denoising steps of ``tedana`` are stable enough
+to integrate into ``FMRIPrep`` and the ```FMRIPrep`` project is updated to process ME-EPI scans.
+
+
+.. _Extensions and Improvements to ME-EPI processing:
+
+Method extensions & improvements
+````````````````````````````````
+**Summary**:
+Overall, each of the listed deliverables will support a broader goal:
+to improve on ME-EPI processing itself.
+This is an important research question and will advance the state-of-the-art in ME-EPI processing.
```

Comment:
Javier suggests that measuring quality assurance here will be an important part of this process. I agree, as noted in https://github.com/ME-ICA/tedana/issues/153#issuecomment-436413069